api/ApiHandler.py

```diff
@@ -1,21 +1,45 @@
-from flask_restful import Api, Resource, request
+from flask_restful import Resource
+import requests;
+import os;
+
+from flask import Flask, jsonify, json
+
 
 class ApiHandler(Resource):
+  
   def get(self):
-    search_term = request.args.get("query")
-    ## TODO: return err if query is invalid twitter handle
-    ## TODO: use search_term to return proper array of words
-    return {
-      'resultStatus': 'SUCCESS',
-      'words': [
-                {
-                    "text": 'told',
-                    "value": 64
-                },
-                {
-                    "text": "hi", 
-                    "value": 1
-                }
-            ]
-      }
+      url = 'https://api.twitter.com/1.1/statuses/user_timeline.json'
+      Twitter_Barear_Token= os.environ.get('Twitter_Barear_Token')
+
+      header = {"Authorization":  f"Bearer {Twitter_Barear_Token}"}
+      userName = "emanWamda" #requests.args.get("searchQuery")
+      params = {'screen_name': userName};
+      response = requests.get(url, headers= header, params = params)
+
+      if response.status_code != 200 :
+          raise Exception(
+              "Request returned an error: {} {}".format(
+              response.status_code, response.text    
+              )
+          )
+
+      print('tweets');
+      tweets = response.json();
+
+      dict = {};
+      for x in tweets: 
+            if x['text'] in dict: 
+                ++dict[x['text']]
+            else:
+               dict[x['text']]= 1
+
+      max_word= ""       
+      max = 0    
+      for key in dict.keys(): 
+        if dict[key] > max:
+            max = dict[key]
+            max_word = key
 
+      print(max)
+      return jsonify({'text': max, 'value': max_word})
```

Comment:
this response expects something like:

```
{
      'resultStatus': 'SUCCESS',
      'words': [
                {
                    "text": 'told',
                    "value": 64
                },
                {
                    "text": "hi", 
                    "value": 1
                }
            ]
      }
      ```
      
      So what we need to do is:
      1.  go through the tweets returned and break them up into words (maybe make each word a key in a dictionary?)
      2. count how many times each word appears (maybe make that the value)
      3. If we want we can exclude little words like "a in on the" etc, but that can be after the MVP

Reply:
In case it is helpful here is an example word cloud: https://www.mentimeter.com/features/word-cloud

So you see how some words in the cloud are bigger? To do that, we need data for how many time each word occurrs!